src/app/zap-templates/zcl/data-model/chip/media-playback-cluster.xml

```diff
@@ -84,16 +84,16 @@ limitations under the License.
 
     <command source="server" code="0x0A" name="PlaybackResponse" optional="false">
       <description>This command SHALL be generated in response to various Playback Request commands.</description>
-      <arg name="status" type="MediaPlaybackStatusEnum"/>
-      <arg name="data"   type="CHAR_STRING" optional="true"/>
+      <arg name="Status" type="MediaPlaybackStatusEnum"/>
+      <arg name="Data"   type="CHAR_STRING" optional="true"/>
```

Comment:
Spec seems to say `OCTET_STRING` is the right thing.

Reply:
That would break backwards compat.  I will figure out what to do here.

Reply:
I filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/6204 on this to get it sorted out.